patches/NDK_3.9/Include/include_h/graphics/gfxmacros.h.diff

```diff
@@ -0,0 +1,22 @@
+--- NDK_3.9/Include/include_h/graphics/gfxmacros.h.orig	2017-01-09 14:37:17.040582249 +0100
++++ NDK_3.9/Include/include_h/graphics/gfxmacros.h	2017-01-09 14:37:32.084233042 +0100
+@@ -55,9 +55,6 @@
+ #define SafeSetOutlinePen(w,c)	  {if (GfxBase->LibNode.lib_Version<39) { (w)->AOlPen = c;(w)->Flags |= AREAOUTLINE;} else SetOutlinePen(w,c); }
+ #define SafeSetWriteMask(w,m)	{if (GfxBase->LibNode.lib_Version<39) { (w)->Mask = (m);} else SetWriteMask(w,m); }
+ 
+-/* synonym for GetOPen for consistency with SetOutlinePen */
+-#define GetOutlinePen(rp) GetOPen(rp)
```

Comment:
Hmm... shouldn't it look as follows:
```c
#define GetOPen(rp) GetOutlinePen(rp)
```

I think someone at Commodore made a mistake. `GetOutlinePen` is defined in many places including `fd/graphics_lib.fd`, `clib/graphics_protos.h`, `pragmas/graphics_pragmas.h` and`amiga.lib`, whereas `GetOPen` is only referenced in the documentation.